docs/docs/guides/manage/self-hosted/production.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Production Checklist
+---
+
+As soon as you successfully deployed ZITADEL as a proof of concept using one of our [deployment guides](/docs/guides/deploy/overview),
+you are ready to configure ZITADEL for production use in your system landscape.
+
+## High Availability
+
+We recommend running ZITADEL higly available using an orchestrator that schedules ZITADEL on multiple nodes, like [Kubernetes](/docs/guides/deploy/kubernetes).
+
+## Binary Configuration
+
+Read [on the configure page](/docs/guides/manage/self-hosted/configure) about the available options you have to configure the ZITADEL binary.
+
+## Networking
+
+- For enabling access via **HTTPS** and restricting access to **HTTPS**, head to [the description of your TLS options](/docs/guides/manage/self-hosted/tls_modes).
+- If you want to front ZITADEL with a reverse proxy, web application firewall or content delivery network, make sure to support **[HTTP/2](/docs/guides/manage/self-hosted/http2)**.
+- You can also refer to some **[example reverse proxy configurations](/docs/guides/manage/self-hosted/reverseproxy/reverse_proxy)**.
+
+## Monitoring
+
+By default, **metrics** are exposed at /debug/metrics in OpenTelemetry (otel) format.
+They can easily be consumed by an [otel collector](https://opentelemetry.io/docs/collector/) and be sent to or scraped by various backends,
+for example [Prometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter).
+
+Also, you can enable **tracing** in the ZITADEL configuration.
+
+```yaml
+Tracing:
+  # Choose one in "otel", "google", "log" and "none"
+  Type: google
+  Fraction: 1
+  MetricPrefix: zitadel
+```
+
+## Database
+
+Depending on your environment, you maybe would want to tweak some settings about how ZITADEL interacts with the database in the Database section of your ZITADEL configuration. Read more about your [database configuration options](/docs/guides/manage/self-hosted/database).
+
+```yaml
+Database:
+  cockroach:
+    Host: localhost
+    Port: 26257
+    Database: zitadel
+    //highlight-start
+    MaxOpenConns: 20
+    MaxConnLifetime: 30m
+    MaxConnIdleTime: 30m
+    //highlight-end
+    Options: ""
+```
+
+You also might want to configure how projections are computed. These are the default values:
```

Comment:
I think we need to point people to a concepts page to explain what projections are

Reply:
Will do 👍